### ISSUE TITLE
Porter adding empty manifest to cache

### ISSUE THREAD

**Issue by MikeLarah** [deislabs/porter#991]
## Describe the bug
When running a command (e.g. `porter explain`) using a `--tag`, Porter adds an empty `porter.yaml` manifest into the Porter cache directory. 

![image](https://user-images.githubusercontent.com/2104606/79458837-4e3e4500-7fea-11ea-837d-458b8136642e.png)

This causes subsequent Porter commands against the same tag to fail with an error like the following:
```
Error: could not resolve bundle: unable to load bundle squillace/functions-java-hello:v0.5.5 from cache: unable to read cached manifest at C:\User
s\MikeLarah\.porter\cache\94cf39026be82eca396eb95299bfc598\porter.yaml: 3 errors occurred:
        * no mixins declared
        * no install action defined
        * no uninstall action defined
```
Note: this is on a Windows machine.

## To Reproduce
1. Run `porter explain --tag squillace/functions-java-hello:v0.5.5`
2.  Run `porter explain --tag squillace/functions-java-hello:v0.5.5` again
3. See error

## Expected behavior
Command should work on second invocation.

## Version
porter v0.26.0-beta.1 (b15c4109)

**Comment by carolynvs** [comment]
Thanks for the reproduction steps. I just verified that the creation of the empty porter.yaml has been fixed on our latest [canary](https://porter.sh/install/#canary) build. So the bad file shouldn't get created in the first place once we cut a new patch release today. 

Normally a `install --force` would cleanup a bad cache entry, but it only does an overwrite, it doesn't remove the entire directory and its contents. Let me see if it's possible to have us clean up after the bad data that was created by the bad release in this patch or if it's going to have to require a manual step of people clearing their cache for now.
